encoding/json/decode.go

```diff
@@ -53,6 +55,15 @@ func WithAllowUnstructuredStaticTypes(allow bool) Option {
 	}
 }
 
+// WithBackwardsCompatability returns a new Decoder Option
```

Comment:
Here and elsewhere: Typo: spelling is compatibility

Reply:
fixed the typo